clients/android/app/src/main/java/app/lockbook/loggedin/mainscreen/MainScreenActivity.kt

```diff
@@ -0,0 +1,43 @@
+package app.lockbook.loggedin.mainscreen
+
+import android.content.Intent
+import android.os.Bundle
+import android.view.Menu
+import android.view.MenuItem
+import androidx.appcompat.app.AppCompatActivity
+import app.lockbook.R
+import app.lockbook.loggedin.settings.SettingsActivity
+
+class MainScreenActivity : AppCompatActivity() {
```

Comment:
This should probably be called `ListFilesActivity` (or something else) `Main` is too ambiguous